Admin/RouteAdmin.php

```diff
@@ -1,227 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony CMF package.
- *
- * (c) 2011-2015 Symfony CMF
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Cmf\Bundle\RoutingBundle\Admin;
-
-use Sonata\AdminBundle\Datagrid\DatagridMapper;
-use Sonata\AdminBundle\Datagrid\ListMapper;
-use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\CoreBundle\Form\Type\ImmutableArrayType;
-use Sonata\DoctrinePHPCRAdminBundle\Admin\Admin;
-use Sonata\DoctrinePHPCRAdminBundle\Form\Type\TreeModelType;
-use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
-use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
-use Symfony\Cmf\Bundle\RoutingBundle\Model\Route;
-use Symfony\Cmf\Bundle\RoutingBundle\Form\Type\RouteTypeType;
-use PHPCR\Util\PathHelper;
-
-class RouteAdmin extends Admin
-{
-    protected $translationDomain = 'CmfRoutingBundle';
-
-    /**
-     * Root path for the route parent selection.
-     *
-     * @var string
-     */
-    protected $routeRoot;
-
-    /**
-     * Root path for the route content selection.
-     *
-     * @var string
-     */
-    protected $contentRoot;
-
-    /**
-     * Full class name for content that can be referenced by a route.
-     *
-     * @var string
-     */
-    protected $contentClass;
-
-    /**
-     * @var ControllerResolverInterface
-     *
-     * @deprecated Since 1.4, use the RouteDefaults validator on your document.
-     */
-    protected $controllerResolver;
-
-    protected function configureListFields(ListMapper $listMapper)
-    {
-        $listMapper->addIdentifier('path', 'text');
-    }
-
-    protected function configureFormFields(FormMapper $formMapper)
-    {
-        $formMapper
-            ->with('form.group_general', array(
-                'translation_domain' => 'CmfRoutingBundle',
-            ))
-                ->add(
-                    'parentDocument',
-                    TreeModelType::class,
-                    array('choice_list' => array(), 'select_root_node' => true, 'root_node' => $this->routeRoot)
-                )
-                ->add('name', TextType::class)
-        ->end();
-
-        if (null === $this->getParentFieldDescription()) {
-            $formMapper
-                ->with('form.group_general', array(
-                    'translation_domain' => 'CmfRoutingBundle',
-                ))
-                    ->add('content', TreeModelType::class, array('choice_list' => array(), 'required' => false, 'root_node' => $this->contentRoot))
-                ->end()
-                ->with('form.group_advanced', array(
-                    'translation_domain' => 'CmfRoutingBundle',
```

Comment:
we should have a form type for the general things and one for the advanced part.